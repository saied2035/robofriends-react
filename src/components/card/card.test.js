import {shallow} from 'enzyme'
import Card from './card'

it("render card component",() => {
    const props= {
        id:1,
        name:'saied',
        email:"saied@gmail.com"
    }
    expect(shallow(<Card {...props}/>)).toMatchSnapshot()
})
