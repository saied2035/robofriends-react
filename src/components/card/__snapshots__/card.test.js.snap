// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render card component 1`] = `
<div
  className="flex justify-center flex-wrap pa2 ma2 br3 bg-light-yellow grow shadow-5"
  style={
    Object {
      "height": "22rem",
      "width": "18rem",
    }
  }
>
  <img
    alt="robot"
    height="200"
    src="https://robohash.org/1?size=400x400"
    width="200"
  />
  <h2>
    saied
  </h2>
  <p>
    saied@gmail.com
  </p>
</div>
`;
